packages/2d/src/components/Polygon.ts

```diff
@@ -0,0 +1,104 @@
+import {Shape, ShapeProps} from './Shape';
+import {initial, signal} from '../decorators';
+import {SignalValue, SimpleSignal} from '@motion-canvas/core/lib/signals';
+
+export interface PolygonProps extends ShapeProps {
+  /**
+   * {@inheritDoc Polygon.nSides}
+   */
+  nSides?: SignalValue<number>;
```

Comment:
What would you say about calling this prop simply `sides`?

Reply:
Sounds good to me.

Reply:
And thanks for the hint on using `<Node>` instead.  I think this works.